integration-tests/src/test/resources/bash-scripts/setup_wdt.sh

```diff
@@ -79,6 +79,9 @@ SCRIPTPATH="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
 WDT_MODEL_FILE=${WDT_MODEL_FILE:-"$SCRIPTPATH/wdt_model.yaml"}
 WDT_VAR_FILE=${WDT_VAR_FILE:-"$SCRIPTPATH/create-domain-inputs.yaml"}
 
+CDIR=`pwd`
+echo "CDIR = $CDIR"
+echo "SCRIPTPATH = $SCRIPTPATH"
```

Comment:
Can you remove the above 3 lines? I added that when trying to debug

Reply:
Done